src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -2039,18 +1832,18 @@ void applyTransactions (
             try
             {
                 switch (applyTransaction (app, view,
-                    it->second, certainRetry, flags, j))
+                    *it->second, certainRetry, tapNO_CHECK_SIGN, j))
```

Comment:
I would say that if a bad transaction got this far, we're already sunk. We've already passed two check -- once when we decided to apply the transaction and once when we tried to apply it and were told to retry it. This only bypasses the standalone, ledger independent, basic crypto checks.